algorithms/branching.py

```diff
@@ -33,7 +32,7 @@ def count_isomorphisms(G: 'Graph', D: 'List[Vertex]', I: 'List[Vertex]', count_f
         return 1
 
     # Get the colors and its vertices of the current graph
-    colors = get_colors(G)
+    colors = G.colors
```

Comment:
Dit hoeft in principe niet meer, alle colors variabelen kunnen worden vervangen door G.colors

Dat maakt het ook leesbaarder in de code.

Reply:
Klopt, heb ik veranderd.